src/main/java/com/xceptance/xlt/engine/dns/XltDnsResolver.java

```diff
@@ -54,6 +58,10 @@
 
     private static final String PROP_RECORD_ADDRESSES = PROP_PREFIX_DNS + "recordAddresses";
 
+    private static final String PROP_IGNORE_IPV4_ADDRESSES = PROP_PREFIX_DNS + "ignoreIPv4Addresses";
+
+    private static final String PROP_IGNORE_IPV6_ADDRESSES = PROP_PREFIX_DNS + "ignoreIPv6Addresses";
+
```

Comment:
What about to use `xlt.dns.ipv4` and `xlt.dns.ipv6` as property names? Having it configured to `false` would then simply mean that this protocol version is disabled, and `true` having it enabled respectively.

Reply:
I like it better the way it is now. With plain "ipv4" and "ipv6" it may not be clear what is meant. I could also be related to the way DNS is contacted, etc. You know what I mean?

Reply:
Roger.